src/recursive_max/__init__.py

```diff
@@ -0,0 +1,7 @@
+def recursive_max(arr):
+    if not arr:
+        return None
```

Comment:
Throwing error is better. What is maximum of nothing?

Reply:
ok, I'll get it